docs/integration/kubernetes.md

```diff
@@ -192,598 +192,593 @@ Container and Kubernetes metrics are collected along with labels and tags, which
 
 ### Storage Metrics
 
-|Name|Type|Unit|Numeric Type|Label|Description|
-|----|----|----|------------|-----|-----------|
-|kubernetes.pvc.available|gauge|bytes|long|available bytes|number of available bytes in the volume|
-|kubernetes.pvc.used|gauge|bytes|long|used bytes|number of used bytes in the volume|
-|kubernetes.pvc.capacity|gauge|bytes|long|volume capacity|the capacity in bytes of the volume|
+|Name|Type|Unit|Numeric Type|Label| Description                           |
+|----|----|----|------------|-----|---------------------------------------|
+|kubernetes.pvc.available|gauge|bytes|long|available bytes|number of available bytes in the volume |
+|kubernetes.pvc.used|gauge|bytes|long|used bytes|number of used bytes in the volume     |
+|kubernetes.pvc.capacity|gauge|bytes|long|volume capacity|the capacity in bytes of the volume    |
 
 ### Kubelet Metrics
 #### Volume Stats
-|Metric Name|Unit|Description|
-|----|----|-----------|
-|pvc.used|int64|Number of used inodes in the volume|
-|pvc.available|int64|Number of available bytes in the volume|
-|pvc.capacity|int64|Capacity in bytes of the volume|
+|Metric Name|Unit| Description                           |
```

Comment:
int64, etc. etc. (see all other tables below) are not "Unit".  "Unit" was meant like "seconds", "bytes", "count" and such.
.... then we don't have to include things like ".... in seconds" in the description....

Reply:
so, I change all the tables with the real unit?

Reply:
@cpipilas ? (out for the rest of the week)

Reply:
We can add `Type` and `Unit`.  
`Type` can include things like gauge, long gauge, counter, etc. 
`Unit` can include bytes, milliseconds, etc.

Reply:
created ticket https://sematext.atlassian.net/browse/SC-17219